html/core/footer_row.go

```diff
@@ -15,9 +16,13 @@ func (c *FooterRow) WriteHTMLTo(w io.Writer) (int64, error) {
 		"https://github.com/elliotchance/gedcom",
 	)
 
+  curTime := time.Now()
+  layout := "Jan 2, 2006, 3.04 pm"
 	content := NewComponents(
 		NewText("Generated with "),
 		link,
+    NewText(" on "),
```

Comment:
If you're going to add this, it needs to be off by default (enabled through a CLI flag) otherwise the output is not deterministic for source control and other syncing.

Also, please run `go fmt` to fix up the indenting.